website/docs/components/stepper/list/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,11 @@
+## Anatomy
+
+![Stepper Navigation Anatomy](/assets/components/stepper/list/stepper-list-anatomy.png =268x*)
```

Comment:
The images should be appropriately sized in Figma before export that you shouldn't need "=268x*"